doc/api/vm.md

```diff
@@ -125,6 +125,10 @@ console.log(util.inspect(sandbox));
 // { animal: 'cat', count: 12, name: 'kitty' }
 ```
 
+*Note*: Using the `timeout` or `breakOnSigint` options will result in new
+event loops and corresponding threads being started, which have a non-zero
```

Comment:
Yeah, `breakOnSigint` doesn’t start its own loop, only an extra thread. It’s definitely just an implementation detail, but if you have suggestions for better wording, I’ll take them. :)